### DISCUSSION TITLE
Advanced Wi-Fi Usage | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#461] [Q&A]
# Advanced Wi-Fi Usage | Seeed Studio Wiki

This wiki introduces some of the advanced library usage of the Wi-Fi such as HTTPClient, DNSServer and WebServer Libraries. With these libraries implemented, you are able to develop your IoT projects with simple APIs.

https://wiki.seeedstudio.com/Wio-Terminal-Advanced-Wi-Fi/

**Comment by 2clchino** [comment]
The mDNS-SD Example source code looks wrong

**Reply by 2clchino** [reply]
The right one is probably this one.
https://github.com/Seeed-Studio/Seeed_Arduino_rpcmDNS/blob/main/examples/mDNS-SD_Extended/mDNS-SD_Extended.ino

**Reply by MatthewJeffson** [reply]
Hi! Thanks for pointing it out. The contents have been changed.
Regards
